app/build.gradle

```diff
@@ -1,26 +1,25 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdk 27
+    compileSdk 33
     ndkVersion "22.1.7171670"
 
-    lint {
-        disable 'ExpiredTargetSdkVersion'
-    }
-
     defaultConfig {
         applicationId "de.tu_darmstadt.seemoo.nfcgate"
         minSdkVersion 19
-        targetSdkVersion 27
-        versionCode 9
-        versionName "2.3.0"
+        targetSdkVersion 33
+        versionCode 10
+        versionName "2.3.1-rc"
```

Comment:
let's not touch the version code and name in this commit

Reply:
Shouldn't it at least be a SNAPSHOT version or something? It's really bad practice to have more than one commit stating an explicit version number.

Reply:
We currently do not have snapshot releases so that every release has a unique version. When we create a new release, we increase versionCode and set a new versionName. For a release we collect multiple changes, so bumping the version not be part of this PR.

Reply:
Done. :)